README.md

```diff
@@ -451,11 +451,11 @@ return_random_cases(ctu05, percent = 0.25, seed = 1337, date = "2019-03-18",
 ```
 
 ```
-##           mnpaid                   centre mnpvisstartdate
-## 1  RPACK-INS-011 Inselspital Bern (RPACK)      2019-04-11
-## 4  RPACK-INS-014 Inselspital Bern (RPACK)      2019-04-14
-## 41 RPACK-CBE-004   Charité Berlin (RPACK)      2019-04-04
-## 3  RPACK-CBE-003   Charité Berlin (RPACK)      2019-04-03
+##          mnpaid                   centre mnpvisstartdate
+## 2 RPACK-INS-012 Inselspital Bern (RPACK)      2019-04-12
+## 4 RPACK-INS-014 Inselspital Bern (RPACK)      2019-04-14
+## 5 RPACK-CBE-005   Charité Berlin (RPACK)      2019-04-05
+## 3 RPACK-CBE-003   Charité Berlin (RPACK)      2019-04-03
 ```
```

Comment:
This is not related directly to your PR but its confusing to me. This should not change.

Any ideas? @markomi @aghaynes

Reply:
different systems (e.g. unix/Windows) use different random number tables I think.. dont know for sure though...
We can check...
On windows...
```
> set.seed(1234)
> rnorm(10)
 [1] -1.2070657  0.2774292  1.0844412 -2.3456977  0.4291247  0.5060559 -0.5747400 -0.5466319 -0.5644520
[10] -0.8900378
```
Can you both run those couple of lines too?

Reply:
```
> set.seed(1234)
> rnorm(10)
 [1] -1.2070657  0.2774292  1.0844412 -2.3456977  0.4291247  0.5060559 -0.5747400
 [8] -0.5466319 -0.5644520 -0.8900378

Reply:
platform       x86_64-pc-linux-gnu 
version.string R version 3.6.1 (2019-07-05)
```
> set.seed(1234)
> rnorm(10)
[1] -1.2070657  0.2774292  1.0844412 -2.3456977  0.4291247  0.5060559
 [7] -0.5747400 -0.5466319 -0.5644520 -0.8900378
```

Reply:
So this is likely due to openSUSE since all results that we posted here are the same and Milica was using an openSUSE OS yesterday.